app/qml/Fonts.qml

```diff
@@ -28,15 +28,15 @@ QtObject{
     property bool lowResolutionFont: _font.lowResolutionFont
 
     property int pixelSize: lowResolutionFont
-                                 ? _font.pixelSize
+                                 ? _font.pixelSize * 2.0 / appSettings.pixelRatio
```

Comment:
With the lowResolutionFont size should be static (like when scanlines are applied), this created some issues in my tests. Does it behave differently in your tests if you revert this line?